### PR TITLE
Add (minimal) static source

### PR DESCRIPTION
**Pull request by carlank**

### CODE REVIEW

File: src/Bot.js [hunk 1]
```diff
@@ -14,18 +17,28 @@ class Bot {
...
     /**
      * Configure a new channel
      * @param  {String}   channel  Channel ID
      * @param  {Number}   delay    Time between bumps in seconds
      * @param  {Function} callback Callback to execute upon revival
+     * @param  {Array}    tags     Tags to considered relevant
      * @throws When refused
      */
-    configure(channel, delay, callback) {
+    configure(channel, delay, callback, tags = []) {
```
Comment (carlank) [inline src/Bot.js:29]: Think the signature is getting big enough to warrant an object?

File: src/Bot.js [hunk 2]
```diff
@@ -14,18 +17,28 @@ class Bot {
...
         if(delay < 10){
             throw `That's too often! Choose a time over 10 seconds.`;
         }
-        this.channels.set(channel, {callback, delay, updated: new Date()});
+        this.channels.set(channel, {callback, delay, updated: new Date(), tags: tags});
```
Comment (carlank) [inline src/Bot.js:33]: This can just be `tags` instead of `tags: tags`

File: src/Bot.js [hunk 3]
```diff
@@ -44,10 +57,27 @@ class Bot {
     reviveChannels(date) {
         this.channels.forEach(channel => {
             if (date.getTime() - channel.updated.getTime() > channel.delay * 1000) {
-                channel.callback();
+                let source = this.chooseSourceFor(channel);
+                channel.callback(source ? source.getMessage() : '');
```
Comment (carlank) [inline src/Bot.js:61]: maybe `source ? source.getMessage() : undefined`?

File: src/Bot.js (hunk 2 above)
Comment (Stratadox) [reply src/Bot.js:33]: Oh nice

File: src/Bot.js (hunk 1 above)
Comment (Stratadox) [reply src/Bot.js:29]: Yea

File: src/Bot.js (hunk 3 above)
Comment (Stratadox) [reply src/Bot.js:61]: Yea I was going to add a default source instead, this is a temporary measure

Comment (carlank) [reply src/Bot.js:61]: 👍

**Pull request merged**